README.md

```diff
@@ -118,41 +118,42 @@ graph TD
 1. Fill in the RFC. Put care into the details: RFCs that do not present
    convincing motivation, demonstrate understanding of the impact of the design,
    or are disingenuous about the drawbacks or alternatives tend to be
-   poorly-received. You might want to create a PR in your fork of the RFCs
-   repository to help you flesh it out with a few supporters or chat/video
-   conference with a few people involved in the topic of the RFC.
-2. In case your RFC is a technical proposal, you might want to prepare a
+   poorly-received. Consider using [Semantic Line Breaks](https://sembr.org/)
+   in oder to get better diffs on later amendments.
+2. Consider publishing your RFC as pre-RFC [in the forum](https://discourse.nixos.org/c/dev/rfc-steering-committee/33)
+   to gather initial feedback and iron out the remaining typos.
+3. In case your RFC is a technical proposal, you might want to prepare a
    prototype of your idea to firstly make yourself aware of potential pitfalls
    and also help reviewers understand the RFC. Code may be able to explain some
    issues in short.
-3. Submit a pull request. As a pull request the RFC will receive design feedback
+4. Submit a pull request. As a pull request the RFC will receive design feedback
    from the larger community, and the author should be prepared to revise it in
    response.
-4. For the nomination process for potential members of the RFC Shepherd Team,
+5. For the nomination process for potential members of the RFC Shepherd Team,
    that is specific to each RFC, anyone interested can either nominate another
    person or themselves to be a potential member of the RFC Shepherd Team. This
    can already be done when submitting the PR.
-5. The RFC Steering Committee assigns a subset of the nominees to the RFC
+6. The RFC Steering Committee assigns a subset of the nominees to the RFC
    Shepherd Team and designates a leader for it. This has to be done
    unanimously.
-6. Build consensus and integrate feedback. RFCs that have broad support are much
+7. Build consensus and integrate feedback. RFCs that have broad support are much
```

Comment:
Why not make it all `1.`? Future changes of order won't have that diff noise.

Reply:
Didn't know that was a thing, but good idea